src/target/spirv/codegen_spirv.cc

```diff
@@ -726,5 +885,51 @@ void CodeGenSPIRV::VisitStmt_(const SeqStmtNode* op) {
 
 void CodeGenSPIRV::VisitStmt_(const EvaluateNode* op) { MakeValue(op->value); }
 
+int CodeGenSPIRV::Stoi(const std::string& str) {
+  try {
+    return std::stoi(str);
+  } catch (std::invalid_argument& e) {
+    LOG(FATAL) << "Cannot convert \"" << str << "\" to int";
+    throw;
+  }
+}
+
+std::pair<int32_t, int32_t> CodeGenSPIRV::GetWmmaFragmentSize(const VarNode* variable) {
```

Comment:
Please don't duplicate this with `codegen_cuda.cc`

Reply:
I factor out this code now.